File: 0070-climbing-stairs/0070-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
        if(n <= 2)
            return n;
        int first = 1;
        int second = 2;
        int num{0};
        
        for(int i = 2; i < n; i++)
        {
            num = first + second;
            first = second;
            second = num;
        }
        return num;
        
    }
};

File: 0070-climbing-stairs/0070-climbing-stairs.go
func climbStairs(n int) int {
    if n < 2 {
        return 1;
    }
    first, second := 1, 1;
    third := 0;
    for n > 1 {
        third = first + second;
        first = second;
        second = third;
        n--;
    }
    return third;
}